udpa/annotations/migrate.proto

```diff
@@ -28,12 +28,20 @@ extend google.protobuf.FileOptions {
 }
 
 message MigrateAnnotation {
-  // Rename the message/field/enum/enum value in next version.
-  // This doesn't handle moving between packages. Use move_to_package in
-  // FileMigrateAnnotation for that.
+  // Rename the message/enum/enum value in next version.
```

Comment:
typo enum/enum

Reply:
It's actually "enum"/"enum value".. not sure how to better phrase, as these are the terms from `DescriptorProto`.

Reply:
Oops got it.